helix-term/src/ui/picker.rs

```diff
@@ -945,7 +945,7 @@ impl<I: 'static + Send + Sync, D: 'static + Send + Sync> Component for Picker<I,
             self.show_preview && self.file_fn.is_some() && area.width > MIN_AREA_WIDTH_FOR_PREVIEW;
 
         let picker_width = if render_preview {
-            area.width / 2
```

Comment:
If possible: Would be nice if you could add a little comment what that `* 3` and `/ 8` means :eyes: